database/src/main/java/com/mint/db/replication/impl/ReplicatedLogManagerImpl.java

```diff
@@ -27,25 +37,32 @@ public class ReplicatedLogManagerImpl implements ReplicatedLogManager<MemorySegm
     private static final int BLOB_BUFFER_SIZE = 512;
     private final NodeConfig nodeConfig;
     private final Path logFile;
-    private final OutputStream outputStream;
+    private final Path indexFile;
     private final ByteArraySegment longBuffer = new ByteArraySegment(Long.BYTES);
     private final ByteArraySegment blobBuffer = new ByteArraySegment(BLOB_BUFFER_SIZE);
-
+    private OutputStream logOutputStream;
+    private OutputStream indexOutputStream;
+    private long lastLogOffset = 0;
     private PersistentState state;
     private long commitIndex = 0; // todo read from file
+    private LogId lastLogId;
```

Comment:
```suggestion
    private LogId lastLogId = new LogId(0, 0);
```

Возможно это пофиксит проблему с тестами

Reply:
да, пофиксило